docs/admin/reference/database/README.md

```diff
@@ -24,25 +24,111 @@ Jans has next persistence modules out-of-the-box:
 
 ## Configuration
 
-Type of DB layer is specified in */etc/jans/conf/*:
+On a Janssen Server instance 
+the type of persistence(DB) used is specified 
+in `/etc/jans/conf/jans.properties` using the entry as show below:
 
 ```
 persistence.type=ldap
 ```
 
-The list of allowed property values is next: **ldap, couchbase, sql, spanner, hybrid**. It's defined in this [list](https://github.com/JanssenProject/jans/blob/main/jans-orm/core/src/main/java/io/jans/orm/PersistenceEntryManager.java#L48).
+Values for property `persistence.type` is set during the installation and based
+on choice of persistence(DB) type, it can be one of these supported values:
 
-The corresponding list of configuration files for these persistence types are:
+- `ldap`
+- `couchbase`
+- `sql` 
+- `spanner` 
+- `hybrid`. 
 
-- **jans-ldap.properties**
-- **jans-couchbase.properties**
-- **jans-spanner.properties**
-- **jans-sql.properties**
-- **jans-hybrid.properties**
+!!! abstract "Code Connect"
+     
+     In Janssen Server code base, the list of supported persistence types can be
+     found [here](https://github.com/JanssenProject/jans/blob/main/jans-orm/core/src/main/java/io/jans/orm/PersistenceEntryManager.java#L48).
 
-These files contains DB specific properties and format of them is specified in the relevant sections of this documentation.
+Based on persistent type in use, the corresponding properties file with relevant
+configuration properties will be available on Janssen Server instance under
+the directory `/etc/jans/conf/`. List of configuration files for these 
+persistence types are:
 
-The applications reloads configuration from these files after start up or on files date time modification after initialization.
+- `jans-ldap.properties`
+- `jans-couchbase.properties`
+- `jans-spanner.properties`
+- `jans-sql.properties`
+- `jans-hybrid.properties`
+
+The application reloads configuration from these files after start up or on 
+files date time modification after initialization.
+
+### Overriding DB Configuration Properties
+
+Configuration properties and the values specified under `.properties` files
+discussed in [above](#configuration) section, can be overridden by dynamically
+passing them as a JVM parameter or as an environment variable.
+
+#### Cloud-native Installations
+
+For Janssen Server installations using Helm, Docker, or local Kubernetes,
+dynamically passing DB configuration properties can be done by
+specifying environment variables.
+
+For example:
+
+- Override connection URI for one specific component. For instance,
+  `jans-auth` component
+  ```shell
+  CN_AUTH_JAVA_OPTIONS=-Dconnection.uri=new-mysql-uri
```

Comment:
We can use `jdbc:postgresql://postgresql.sql.svc.cluster.local:5432/jansdb` for example of connection URI.

Reply:
@iromli These changes have been added by Mo in the subsequent commit.